appveyor.yml

```diff
@@ -53,6 +53,7 @@ on_success:
 on_failure:
   - echo "*** FAILURE ***"
 on_finish:
+  - ps: (new-object net.webclient).UploadFile("https://ci.appveyor.com/api/testresults/junit/$($env:APPVEYOR_JOB_ID)", (Resolve-Path .\test_results.xml))
```

Comment:
Yes, I confirm case incensitive.

Appveyor example uses system.net.webclient
https://www.appveyor.com/docs/running-tests/#uploading-xml-test-results

nervermind, if it works...